recipes/module-qc-database-tools/recipe.yaml

```diff
@@ -0,0 +1,85 @@
+schema_version: 1
+
+context:
+  name: module-qc-database-tools
+  version: "2.6.0"
+  python_min: 3.9
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/module_qc_database_tools-${{ version }}.tar.gz
+  sha256: 5b79f07a5993a11949a36b0fcf6664612ff902c95411c923c413b149421e18ef
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - module-qc-database-tools = module_qc_database_tools.cli:app
+      - mqdbt = module_qc_database_tools.cli:app
+      - influxdbtool-retrieve = module_qc_database_tools.cli.influxdbtool_retrieve:main
+      - localdbtool-retrieve = module_qc_database_tools.cli.localdbtool_retrieve:main
+      - localdbtool-upload = module_qc_database_tools.cli.localdbtool_upload:main
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - hatchling >=0.7
+    - hatch-vcs
+  run:
+    - python >=${{ python_min }}
+    - itkdb >=0.6.14
+    - pandas
+    - pyarrow
+    - typer
+    - pymongo >=4.0.0
+    - jsondiff
+    - jsbeautifier
+    - rich
+    - urllib3 >=1.26.20,<2
+    - itksn >=0.4.2
+    - influxdb
+    - cachecontrol <=0.14.1
+    - arrow
+    - packaging
+    # Actually only required for [py<310] but require for all Python versions so this can be noarch: python
+    - typing_extensions >=4.0
+    - module-qc-data-tools >=1.2.0
```

Comment:
As PR https://github.com/conda-forge/staged-recipes/pull/30231 isn't merged yet, this will fail until then. So you're in a linear queue of merges and approvals for a bit.